### DISCUSSION TITLE
Run HTTPS with own certificates (not via traefik)

### DISCUSSION THREAD

**Discussion by maquino1985** [fastapi/full-stack-fastapi-template#1807] [Questions]
Deploying in k8s and need to manage my own certs. How do you configure uvicorn to use your own certificates for HTTPS when deploying this app? The docker container automatically runs uvicorn but there is no config file, how can I configure the uvicorn settings?

**Comment by tulanglaoli** [comment]
i have same problem,  not work with own certificates

**Comment by Doneowth** [comment]
@maquino1985  Can you share your k8s deployment helm? Thanks.
